package.json

```diff
@@ -98,7 +98,8 @@
           "modify-customer-orders",
           "multiserver-db-sessioncache",
           "vendure-order-client",
-          "all-plugins"
+          "all-plugins",
+          "vendure-plugin-primary-collection"
```

Comment:
We can change this to `primary-collection`. It's used in commit scopes: E.g. `git commit 'feat(primary-collection): initial commit'`. And the `vendure-plugin-*` prefix makes it a bit tedious to do every time you commit :-)